demo/src/webhooks/subscriptionCreatedOrUpdated.ts

```diff
@@ -69,6 +69,7 @@ export const subscriptionCreatedOrUpdated = async (args) => {
         payload.logger.info(`- Subscription already exists, now updating.`);
         // update existing subscription
         subscriptions[indexOfSubscription] = {
+          stripeSubscriptionID: eventID,
```

Comment:
The `eventID` is not the same as `stripeSubscriptionID`, which is probably what we want to use here.